doc/ceph-volume/drivegroups/README.rst

```diff
@@ -0,0 +1,155 @@
+.. _ceph-volume-drivegroups:
+
+``Concept``
+============
+
+Creating OSDs in ceph is a mostly manual process. DriveGroups are aiming to change that.
+
+Instead of targeting drives individually we try to describe them by using attributes.
+
+.. note:: Valid attributes are `vendor`, `model`, `size`, `rotational`
+          (Attributes are compared to what ceph-volume inventory --format json-pretty reports)
+
+
+
+``Example``
+===========
+
+Imagine a node with 10HDDs and 2SSDs.
+
+HDD:
+
+- Vendor: Example_vendor_A
+- Model: Slow_rusty_spinner_123
+- Size: 2TB
+
+SSD:
+
+- Vendor: Example_vendor_B
+- Model: Fast_thing_987
+- Size: 512GB
+
+
+We'd like to have the two SSDs as dedicated DB/WAL devices.
+
+In a traditional deployment procedure we'd find the device names of these drives and pass them
+to ceph-volume (given we follow the manual deployment process)::
+
+:: ref to ceph-volume docs
+
+With DriveGroups we just create one yaml file::
+
+
+    touch example_drive_groups.yaml
+
+
+Now we add try to describe these drives using their attributes.
+
+The easiest way to differentiate the drives is by their ``rotational`` flag.
+Hence our DriveGroups would look as simple as:
+
+
+.. code-block:: yaml
+
+        default:
+            data_devices:
+              rotational: 0
+            db_devices:
+              rotational: 1
+
+
+If for whatever reason we can't rely on the rotational flag, we may just use any other property.
+
+
+.. code-block:: yaml
+
+        default:
+            data_devices:
+              size: 2TB # high:low notation is also implemented (:N, N:, MIN:MAX)
+            db_devices:
+              size: 512MB # MB or M is fine
```

Comment:
Should be 512GB

Reply:
thanks! please don't review individual commits at this point